protos/local/temporal/sdk/core/activity_task/activity_task.proto

```diff
@@ -75,6 +75,8 @@ enum ActivityCancelReason {
     CANCELLED = 1;
     /// Activity timed out
     TIMED_OUT = 2;
+    /// Core is shutting down and the graceful timeout has elapsed
+    GRACEFUL_SHUTDOWN = 3;
```

Comment:
Would help TS if we kept the reason name: https://github.com/temporalio/sdk-typescript/blob/87e6a158f2623b89ed829c94f6b825eb61dd9d99/packages/worker/src/activity.ts#L22

Reply:
Will do